projects/plugins/beta/class-jetpack-beta.php

```diff
@@ -1094,6 +1094,11 @@ public static function should_update_stable_version() {
 			return false;
 		}
 
+		// Check if running in a docker instance.
```

Comment:
Could we move this up to go with the `IS_PRESSABLE` check above? It would be more performant to check the constant before the file check or the tag check.

Reply:
Yup! Should be moved now.